ports/stm32/boards/NUCLEO_WB55/rfcore_firmware.py

```diff
@@ -558,7 +648,19 @@ def resume():
 
 # Start a firmware update.
 # This will immediately trigger a reset and start the update process on boot.
-def check_for_updates():
-    log("Starting firmware update")
-    _write_state(_STATE_WAITING_FOR_FUS)
-    machine.reset()
+def check_for_updates(force=False):
```

Comment:
Could you be doing a FUS-only update? In which case checking the WS might prevent that from running (unless `force` is set)?

Reply:
Good point. It "feels" unlikely to me that there would be a board in a state where it has up to date WUS but older FUS, but that's certainly possible. I should check for FUS version as well perhaps